apps/easypid/src/hooks/useShouldUsePinForPresentation.tsx

```diff
@@ -1,4 +1,4 @@
-import type { CredentialsForProofRequest } from '@package/agent'
+import type { CredentialsForProofRequest } from '@paradym/wallet-sdk/openid4vc/getCredentialsForProofRequest'
```

Comment:
should we export from root?

Reply:
Yeah I want to fix the exports all in the end. Bit difficult to see what to export now and what not to, as some things still have to be made simpler.